lib/ActionClient.js

```diff
@@ -31,28 +31,33 @@ class ActionClient extends EventEmitter {
     const nh = options.nh;
 
     // FIXME: support user options for these parameters
+    const goalOptions = Object.assign({ queueSize: 10, latching: true }, options.goal);
```

Comment:
yeah - I went and checked with the `roscpp` implementation again for default queue size

https://github.com/ros/actionlib/blob/indigo-devel/include/actionlib/client/action_client.h#L214